### PR TITLE
Fix:#2442 positions is requested twice when loading Holdings

### PR DESCRIPTION
**Pull request by Fullstack-Arshad**
Fix: #2442

### CODE REVIEW

**Comment by Fullstack-Arshad** [comment]
what I found was update function was making that API call and it was being called once from the constructor and once from the ngOninit, removing the removing the API call withing ngOnint was crashing the API call, hence I decided to remove the one in the constructor...

**Comment by dtslvr** [comment]
Thanks for your investigations @Fullstack-Arshad.

I have tested it, but it does not seem to work like this: When you click the `gf-toggle` component (e.g. `Today` or `Max`) now, there is no further request to reload the holdings with the date range parameter. This happened before because the user settings were changed with the click.

File: apps/client/src/app/components/home-holdings/home-holdings.component.ts [hunk 1]
```diff
@@ -66,8 +66,6 @@ export class HomeHoldingsComponent implements OnDestroy, OnInit {
             this.user.permissions,
             permissions.createOrder
           );
-
-          this.update();
```
Comment (dtslvr) [inline apps/client/src/app/components/home-holdings/home-holdings.component.ts:70]: Without it, holdings are not reloaded when you change the date range.

**Review by dtslvr — CHANGES_REQUESTED** [review]

File: apps/client/src/app/components/home-holdings/home-holdings.component.ts (hunk 1 above)
Comment (Fullstack-Arshad) [reply apps/client/src/app/components/home-holdings/home-holdings.component.ts:70]: Thanks for this information @dtslvr, I will further dig into this problem and try to resolve it. But I work in a company and this week I will be busy, therefore will try to do it in my free time.

Comment (Fullstack-Arshad) [reply apps/client/src/app/components/home-holdings/home-holdings.component.ts]: Hi @dtslvr, hope you are doing well, I have added a commit, this should hopefully fix the issue, attaching a video for your reference

https://github.com/ghostfolio/ghostfolio/assets/61023134/4d0774be-065a-4c94-83eb-cdfd292a4425

please let me know if there is still any issue....
Thanks 🙂

**Comment by Fullstack-Arshad** [comment]
Hi @dtslvr , yesterday I added one commit that should hopefully fix this bug. Please check once and let me know if anything else is required, attaching a video clip for your reference

https://github.com/ghostfolio/ghostfolio/assets/61023134/dcd1eedc-f19a-4a59-8383-5691e21867fc

Thank you 🙂

**Review by dtslvr — APPROVED** [review]
Great that you solved the problem, @Fullstack-Arshad! 🙌🏻

**Pull request merged**